github/ci/prow-deploy/molecule/default/prepare.yml

```diff
@@ -1,15 +1,87 @@
-- hosts: localhost
+- hosts: instance
   tasks:
     - set_fact:
         deploy_environment: '{{ lookup("env", "DEPLOY_ENVIRONMENT") }}'
-    - fail:
+    - name: Check DEPLOY_ENVIRONMENT env var
+      fail:
         msg: DEPLOY_ENVIRONMENT must be set
       when: deploy_environment is undefined or deploy_environment == ""
 
+    - set_fact:
+        local_project_infra_root: '{{ playbook_dir}}/../../../../../'
+    - set_fact:
+        project_infra_root: /workspace/repos/project-infra-master
+
+    - name: create project-infra dir
+      file:
+        path: '{{ project_infra_root }}'
+        state: directory
+
+    - name: upload project-infra files
+      synchronize:
+        src: '{{ local_project_infra_root }}'
+        dest: '{{ project_infra_root }}'
+        rsync_opts:
+          - "--exclude=.git"
+          - "--exclude=venv"
+
+    - set_fact:
+        secrets_dir: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/environments/{{ deploy_environment }}/secrets/'
```

Comment:
Beware, this directory is initially completely empty. It's marked as IGNORED in git. Secrets are meant to be copied from the secrets-boilerplate dir.
I see you're creating some secrets here, not sure if you're also creating all the rest.

Reply:
Exactly, some of them are created here, the rest are hardcoded in the kustomize config for the test environment.